src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -295,15 +295,24 @@ local_zone_id | | See (3) below
 3. The `StopPointInJourneyPattern/ScheduledStopPointRef/@ref` attribute is searched in the `PassengerStopAssignment/ScheduledStopPointRef/@ref` attribute of all the `PassengerStopAssignment` nodes of the file
 4. The `PassengerStopAssignment/QuayRef/@ref` is the stop_id of the stop_point (with a prefix).
 
-(2) Definition of pickup_type and drop_off_type:
+(2) Passing midnight
+
+`TimetabledPassingTime/ArrivalTime` can be greater than `TimetabledPassingTime/DepartureTime`. That means has passes midnight and is therefore the next day.
+In this case the same rule as above is applied but 1 day more is added to `TimetabledPassingTime/DepartureTime`.
+
+Eg: `TimetabledPassingTime/ArrivalTime` = "23:50:00", `TimetabledPassingTime/DepartureTime` = "00:10:00",  `TimetabledPassingTime/DepartureDayOffset` = 1
+
+arrival_time = "47:50:00" and departure_time = "48:10:00"
```

Comment:
No. With this case:
`TimetabledPassingTime/ArrivalTime` = "23:50:00", 
`TimetabledPassingTime/DepartureTime` = "00:10:00",  
`TimetabledPassingTime/DepartureDayOffset` = 1

arrival_time is "23:50:00" and departure_time = "24:10:00"